src/jit/gentree.cpp

```diff
@@ -1791,6 +1791,11 @@ regMaskTP GenTreeCall::GetOtherRegMask() const
     return resultMask;
 }
 
+bool GenTreeCall::IsPure(Compiler* compiler) const
+{
```

Comment:
The to-be-written method header should describe what we mean here by Pure (and possibly mention that it is the definition shared by the HelperCallProperties).

---
In reply to: [80796498](https://github.com/dotnet/coreclr/pull/7368#discussion_r80796498) [](ancestors = 80796498)